force-app/main/default/classes/SOQL.cls

```diff
@@ -183,7 +245,12 @@ public with sharing class SOQL {
      * @return A list of Opportunities with an Amount greater than 100,000, ordered by CloseDate, and limited to 5 records.
      */
     public static List<Opportunity> getHighValueOpportunities() {
-        return null; // Replace null with the variable you used to store the result
+        List<Opportunity> highOpps =[SELECT Name, Amount, CloseDate
+        FROM Opportunity
+        WHERE Amount > 100000
+        ORDER BY CloseDate ASC
+        LIMIT 5];
+        return highOpps; // Replace null with the variable you used to store the result
```

Comment:
[I see that you change the style of your queries. 
Are you using a global [snippet ](https://code.visualstudio.com/docs/editor/userdefinedsnippets#:~:text=To%20create%20or%20edit%20your,should%20appear%20for%20all%20languages.)for these, if you are great!

Reply:
I have never used snippets, but I think going forward I will write my queries that way to make them easier to read.